src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -33,25 +33,34 @@ class AptitudePackageManager(PackageManager):
     def __init__(self, env_layer, execution_config, composite_logger, telemetry_writer, status_handler):
         super(AptitudePackageManager, self).__init__(env_layer, execution_config, composite_logger, telemetry_writer, status_handler)
 
-        # Accept EULA (End User License Agreement) as per the EULA settings set by user
-        optional_accept_eula_in_cmd = "ACCEPT_EULA=Y" if execution_config.accept_package_eula else ""
+        # Apt constants config
+        self.APT_SOURCES_LIST_PATH = '/etc/apt/sources.list'
+        self.APT_SOURCES_LIST_DIR_PATH = '/etc/apt/sources.list.d/'
+        self.APT_SOURCES_LIST_DIR_LIST_EXT = 'list'
+        self.APT_SOURCES_LIST_DIR_SRC_EXT = 'sources'
```

Comment:
nit: rename these to self.APT_SOURCES_DIR_LIST_EXT and self.APT_SOURCES_DIR_SRC_EXT, makes it easier to read and follow too

Reply:
All 3 flow from APT_**SOURCES_LIST_DIR**_PATH = '/etc/apt/**sources.list.d**/'

It'd make sense if SOURCES_LIST_DIR was replaced by SOURCES_DIR everywhere (i.e. all three changed). Is that the proposal? The only thing against that is the directory name is sources list dir.

Reply:
**[Candidate for resolving]** Addressed as discussed offline.